sockeye/data_io.py

```diff
@@ -481,14 +481,14 @@ def __init__(self,
         # operations that produce shapes larger than the default bucket size.  In these cases, MXNet
         # will silently allocate additional memory.
         self.provide_data = [
-            mx.io.DataDesc(name=source_data_name,
+            mx.io.DataDesc(name=self.source_data_name,
```

Comment:
fine with me, but any reason why this is important?

Reply:
It's not important. I just made it consistent with the use of all other parameters in __init__ (i.e. use self.*).